src/responsibleai/component_rai_insights.yaml

```diff
@@ -33,6 +33,9 @@ inputs:
   classes:
     type: string # Optional[List[str]]
     default: '[]'
+  use_model_dependency:
```

Comment:
it may be preferable to use a enum for this option so that we don't break user if we support additional modes in dealing with model with different environment.

for example, we can have 

['None', 'local', 'conda']

which correspond to
None: No handling
local: install dependencies as specified in mlflow metadata
conda: install dependencies in separate environment (use inference endpoint model, which could be using http, grpc or other communication protocol to retrieve results from predict or predict_proba)

Reply:
Another option / additional options would be to introduce new arguments asking user to specify dependencies source. User can choose from enums ["mlflow", "custom"] where custom refers to using a custom dependencies list. 

This can be useful in following scenarios:
- pip install in local environment fails for mlflow dependencies. A smaller set of dependencies may be specified instead.
- mlflow dependency detection is not perfect. In the case where unneeded dependency is introduced, or in the case where real dependency are missing, it could be specified by user.